governance/GOVERNANCE.md

```diff
@@ -36,7 +36,7 @@ To reach a decision on an issue or proposal, the proponents must seek maintainer
 
 ## Review Criteria for OSPS Baseline Repository
 
-Any changes intended to be merged in the OSPS Baseline repository shall meet the following minimal criteria:
+Any changes intended to be merged in the OSPS Baseline repository shall meet the following minimal criterion:
```

Comment:
```suggestion
Any changes intended to be merged in the OSPS Baseline repository shall meet the following minimal criteria:
```
"criteria" was correct here, since we list more than one criterion.